statsd/statsd.go

```diff
@@ -456,22 +458,24 @@ func newWithWriter(w io.WriteCloser, o *Options, writerName string) (*Client, er
 		initContainerID(o.containerID, isOriginDetectionEnabled(o, hasEntityID))
 	}
 
+	isUDS := writerName == writerNameUDS || writerName == writerNameUDSDatagram || writerName == writerNameUdsStream
+
 	if o.maxBytesPerPayload == 0 {
-		if writerName == writerNameUDS {
+		if isUDS {
```

Comment:
Commit or PR message should mention this change to payload size as well.

Reply:
🤔 this doesn't change anything no ? before `writerNameUDS` was used for `unix://` / `unixgram://` / `unixstream://`, now because we separate the writer name based on the protocol schema, we intoduce the `isUDS` variable. But nothing else changes

Reply:
Updated the descrption to make this more obvious